azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -15,7 +15,9 @@
  */
 package azkaban.executor;
 
+import azkaban.sla.SlaOption;
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
probably this import is not used. You can install the new style and run it on your changed files only. It may fix some of these cosmetic issues.